### ISSUE TITLE
[I18N] Translate strings

### ISSUE THREAD

**Issue by GOVINDDIXIT** [RocketChat/Rocket.Chat.Android#2128]
## Describe the chore

This is a parent issue. The contributors can choose their native language and help in translating it and updating the strings

#### Open for all
While submitting PR for translations mention partial solving to this issue in your PR description: `Partly solves #2128`.

**Comment by GOVINDDIXIT** [comment]
@filipedelimabrito this should be open

**Comment by philipbrito** [comment]
@GOVINDDIXIT It was closed when merging #2129. Checking the PR's description there (Partial Fix #2128) caused it to be closed.

**Comment by GOVINDDIXIT** [comment]
Yes, @filipedelimabrito  Thanks for updating the issue description  ☺️
